recipes/hercules/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "hercules" %}
+{% set version = "10.1.0" %}
+{% set download_url = "https://github.com/src-d/hercules/releases/download" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: {{ version }}
+
+source:
+  fn: hercules.gz  # [unix]
+  url: {{ download_url }}/v{{ version }}/{{ name }}.linux_amd64.gz          # [linux]
+  sha256: 47821f0b9c3e23e4cea6caf029c90c7649abe9a100d25666107aa11f1b0efbd1  # [linux]
+
+  url: {{ download_url }}/v{{ version }}/{{ name }}.darwin_amd64.gz         # [osx]
+  sha256: 4861dd2fcfd1ec04982befb25f62ccc3eb5baa217d99a834bb8f039636488d94  # [osx]
+
+  url: {{ download_url }}/v{{ version }}/{{ name }}.win64.zip               # [win]
+  sha256: dcd5face27d300ce2062cf84fb44f6c3fb04a3b981fd34c516f55a713afb9ecf  # [win]
+
+build:
+  number: 0
+  script:
+    - gunzip hercules.gz               # [unix]
+    - mkdir -p "$PREFIX/bin"           # [unix]
+    - install hercules "$PREFIX/bin/"  # [unix]
+    - mv hercules.exe "%LIBRARY_BIN%\hercules.exe"  # [win]
+
+requirements:
+  build:
+    # Dummy requirement to ensure conda-forge builds this correctly
+    - make
```

Comment:
if you do not need anything just let it empty as
```
build: []
```

Reply:
@marcelotrevisani Seems not to work. See last commit... There already is an issue https://github.com/conda-forge/staged-recipes/issues/8377

Reply:
Ok. Added a revert commit.